dev_ops/sql/Tables/CreateVotes.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE votes (
+    id BIGSERIAL PRIMARY KEY,
+	poll_id BIGINT REFERENCES polls (id),
+	voter_id BIGINT REFERENCES users (id),
+	vote BOOLEAN NOT NULL, -- True represents the vote that the streamer completed the contract
```

Comment:
If a variable name needs a comment, its not a good variable name. Lets rename this to streamer_flagged_complete or something like that so we feel its descriptive enough to not need a comment.